packages/react/guides/0-Migration-Guide.md

```diff
@@ -2,6 +2,161 @@
 title: Migration guide
 ---
 
+## Migrating to @sanity/sdk-react@1.0.0
+
+### Breaking Changes
+
+1. `useManageFavorite`, `useNavigateToStudioDocument`, and `useRecordDocumentHistoryEvent` now all require a Suspense boundary.
```

Comment:
i'm not sure if we necessarily need to say these require a suspense boundary but rather that they all suspend now. for example, there are cases where it may be better to rely a parent suspense boundary where potentially a `useProjection` would also suspend too and both of those hooks could share the same loading fallback. additionally the SDK now forces in one top-level boundary